pt-BR/mail/chrome/messenger/am-copies.dtd

```diff
@@ -5,40 +5,40 @@
 <!ENTITY fccMailFolder.accesskey "F">
 <!ENTITY fccReplyFollowsParent.label "Colocar respostas na pasta da mensagem que foi respondida">
 <!ENTITY fccReplyFollowsParent.accesskey "C">
-<!ENTITY chooseFolderButton.label "Choose Folder...">
+<!ENTITY chooseFolderButton.label "Escolher pasta...">
 <!-- LOCALIZATION NOTE (bccAddress.label): do not translate "Bcc" in below line -->
 <!ENTITY bccAddress.label "Cco para estes endereços:">
```

Comment:
@brunotiussi Thanks for replying, understood! However, Sherman's instructions are to keep "**Bcc**".